angular/src/app/modules/delivery-management/delivery/request-resource-tab/request-resource-tab.component.ts

```diff
@@ -318,38 +333,48 @@ export class RequestResourceTabComponent
   public updateResourceRequestCV(resourceRequestCV: ResourceRequestCVDto, request: RequestResourceDto): void {  
     const formattedCV = this.formatInterviewDateToVN(resourceRequestCV);
     this.resourceRequestService.updateResourceRequestCV(formattedCV).pipe(
-      catchError(error => {
-        console.error('Error in updateResourceRequestCV API call:', error);
-        return of(null); 
-      })
-    ).subscribe(
-      response => {
+        catchError(error => {
+            console.error('Error in updateResourceRequestCV API call:', error);
+            return of(null); 
+        })
+    ).subscribe(response => {
         if (response && response.success) {
```

Comment:
Gọi nhiều api ở hàm này quá -> gọi 1 thôi, xly hết ở BE
BE có thể tạo ra nhiều api để update riêng cho từng field
khi update status sang Pass -> cũng xly ở BE chứ ko xly theo kiểu fe gọi thêm api